website-components-playground/src/pages/custom-form-modal-page.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { Formik } from 'formik';
+import { CustomFormModalPage } from '@commercetools-frontend/application-components';
+import TextInput from '@commercetools-uikit/text-input';
+import TextField from '@commercetools-uikit/text-field';
+import LayoutApp from '../layouts/layout-app';
+import PlaygroundController from '../components/playground-controller';
+import ModalController from '../components/modal-controller';
+
+const containerId = 'custom-form-modal-page';
+
+const CustomFormModalPageExample = () => (
+  <LayoutApp>
+    <PlaygroundController
+      knobs={[
+        {
+          kind: 'text',
+          name: 'title',
+          label: 'Title',
+          initialValue:
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+        },
+        {
+          kind: 'text',
+          name: 'subtitle',
+          label: 'Subtitle',
+          initialValue:
+            'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+        },
+        {
+          kind: 'select',
+          name: 'hideControls',
+          label: 'Hide Controls?',
+          valueOptions: [
+            { value: false, label: 'No' },
+            { value: true, label: 'Yes' },
+          ],
+          initialValue: false,
+        },
+      ]}
+    >
+      {({ values }) => (
+        <ModalController
+          title="Open the Custom Form Modal Page by clicking on the button"
+          buttonLabel="Open Custom Form Modal Page"
+          containerId={containerId}
+        >
+          {({ isOpen, setIsOpen }) => (
+            <Formik
```

Comment:
If you want you can also useFormik here to simplify things. Doesn't matter though or we do it in one go for all examples in the app.